Gemfile

```diff
@@ -55,6 +55,8 @@ gem 'activeadmin', github: 'activeadmin'
 
 # for handling file uploads
 gem 'carrierwave'
+gem 'fog', require: false
+gem 'mini_magick'
```

Comment:
I think this is needed for image manipulation. If I do not want image manipulation then I think I can comment it out. Right ?